src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,38 @@
+import { fetchArticles }  from "../api";
+import { useEffect, useState } from "react";
+
+export default function ArticlesList() {
+    const [articles, setItems] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    
+    useEffect(() => {
+        fetchArticles().then((articles) => {
```

Comment:
setIsLoading(true) first. Or it might never be true and your page will constantly running the code when isLoading is false